library/core/src/clone.rs

```diff
@@ -185,7 +185,7 @@ mod impls {
                 #[stable(feature = "rust1", since = "1.0.0")]
                 #[rustc_const_unstable(feature = "const_clone", issue = "91805")]
                 impl const Clone for $t {
-                    #[inline]
+                    #[inline(always)]
```

Comment:
This probably does less than you'd expect, because of https://github.com/rust-lang/rust/blob/0f0d5d716a00c1f29cfbf240e12bdccc4f9b9a94/compiler/rustc_mir_transform/src/instcombine.rs#L138